src/redux/missions/missionsSlice.js

```diff
@@ -52,6 +52,13 @@ const missionsSlice = createSlice({
 
         const missionsList = [];
         data.forEach((element) => {
+          // const { mission_id, mission_name, description } = element;
+          // missionsList.push({
+          //   mission_id,
+          //   mission_name,
+          //   description,
+          //   joined: false,
+          // });
```

Comment:
- Removing all comment before you can merge your work is highly recommended.

Reply:
### Fixed.